subsystems/subprojects.settings.gradle

```
@@ -2,15 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // This magically allows subdirs in this subproject to themselves become sub-subprojects in a proper tree structure
-new File(rootDir, 'subsystems').eachDir { possibleSubprojectDir ->
+new File(rootDir, "subsystems").eachDir { possibleSubprojectDir ->
     if (!possibleSubprojectDir.name.startsWith(".")) {
-        def subprojectName = 'subsystems:' + possibleSubprojectDir.name
-        logger.info("Including '$subprojectName' as a sub-project")
-        include subprojectName
-        def subprojectPath = ':' + subprojectName
+        def subprojectName = "subsystems:" + possibleSubprojectDir.name
+        logger.info("Including {} as a sub-project.", subprojectName)
+        include(subprojectName)
+        def subprojectPath = ":" + subprojectName
         def subproject = project(subprojectPath)
         subproject.projectDir = possibleSubprojectDir
     } else {
-        logger.info("Ignoring hidden folder '$possibleSubprojectDir'")
+        logger.info("Ignoring hidden folder {}.", possibleSubprojectDir)
```

```suggestion
        logger.info("Ignoring hidden folder '{}'.", possibleSubprojectDir)
```

It helps sometimes to see quotation marks around file names to visualise any unexpected spaces in them.